raiden/network/transport/matrix/client.py

```diff
@@ -402,6 +402,8 @@ def call(callback, *args, **kwargs):
 
     def _sync(self, timeout_ms=30000):
         """ Reimplements MatrixClient._sync, add 'account_data' support to /sync """
+        log.debug("Sync called", current_user=self.user_id)
+
```

Comment:
I would log the home server here as well

Reply:
where is that information available?